libbeat/processors/add_cloud_metadata/provider_hetzner_cloud_test.go

```diff
@@ -60,6 +69,7 @@ func TestRetrieveHetznerMetadata(t *testing.T) {
 	defer server.Close()
 
 	config, err := conf.NewConfigFrom(map[string]interface{}{
+		// "providers": []string{"hetzner"},
```

Comment:
@axw Thanks for pointing out the config `os.Setenv("AWS_EC2_METADATA_DISABLED", "true")`! Without specifying the `providers` to be `hetzner`, the test passes too now,. Should I still give a `providers` just to be sure? 😂

Reply:
I think it may be better to leave it out, to try and avoid issues like https://github.com/elastic/beats/issues/13816 as much as possible.